packages/web-sdk/src/transports/otlp/payload/transform/types.ts

```diff
@@ -20,21 +15,36 @@ export interface LogRecord {
   traceId?: string;
   spanId?: string;
 }
+export interface ScopeLog {
+  scope: { name: string; version: string };
```

Comment:
Any interest in making an interface for this? I could go either way - but just wondering if you personally have a minimum number of fields required before making an explicit type.

Reply:
MUch better. Let me make it interface.